### DISCUSSION TITLE
Numbered list items should change based on indentation

### DISCUSSION THREAD

**Discussion by danielrearden** [TypeCellOS/BlockNote#2485] [Ideas & Enhancements]
**Is your feature request related to a problem? Please describe.**
Right now, numbered lists use 1, 2, 3, etc. regardless of the level of indentation. In most other apps (Notion, Google Docs, etc.) the characters used for each list item vary by indentation level.

**Describe the solution you'd like**
I would expect numbered lists to look like this

<img width="321" alt="Screenshot 2024-10-30 at 12 47 42 PM" src="https://github.com/user-attachments/assets/f4307289-f2e5-479d-9762-f96967887786">

**Describe alternatives you've considered**
We've looked into implement our own block types as a workaround, but this seems like something that should be part of Blocknote's core functionality.

**Additional context**
N/A

**Bonus**
- [X] I'm a [sponsor](https://github.com/sponsors/YousefED) and would appreciate if you could look into this sooner than later 💖

**Comment by woutervroege** [comment]
Closing this as its a duplicate of https://github.com/TypeCellOS/BlockNote/discussions/2496
